src/embezzler.ts

```diff
@@ -568,6 +570,25 @@ export const gregFights = (
   fightsProp: NumericProperty,
   totalCharges: () => number,
 ) => {
+  function runGregFight(options: EmbezzlerFightRunOptions) {
+    const run = ltbRun();
+    run.constraints.preparation?.();
+    const bunnyBanish = [...getBanishedMonsters().entries()].find(
```

Comment:
I'd change the name `bunnyBanish` to be something more explicit like `priorBunnyBanish`.